apps/web/src/services/cost-explorer/CostExplorerLNB.vue

```diff
@@ -49,13 +80,44 @@ export default {
                     },
                 ], store.getters['user/pagePermissionList']),
             ]),
-            selectedMenu: {} as LNBItem,
         });
 
+        const relocateNotificationState = reactive({
+            isShow: true,
+            data: computed<LNBItem>(() => ({
+                type: 'item',
+                id: MENU_ID.DASHBOARDS,
+                label: 'Go to Dashboard',
```

Comment:
Suggestion: It seems like there might be a need for translation. It would be helpful to leave a comment about it.